extensions/http/jersey/src/main/java/org/eclipse/dataspaceconnector/extension/jersey/JerseyRestService.java

```diff
@@ -49,34 +51,53 @@ public JerseyRestService(JettyService jettyService, TypeManager typeManager, Cor
 
     @Override
     public void registerController(Object controller) {
```

Comment:
This is probably just here to be backwards-compatible, right? In that case it could be @Deprecated

Reply:
Never mind, saw it below.